core/domain/suggestion_registry.py

```diff
@@ -38,9 +38,12 @@
 from core.domain import state_domain
 from core.domain import user_services
 from core.platform import models
+from extensions import domain
 
 from typing import (
-    Any, Callable, Dict, List, Mapping, Optional, Set, Type, TypedDict, Union)
+    Any, Callable, Dict, List, Mapping, Optional, Set, Type, TypedDict, Union,
```

Comment:
Can we not use Any?

Reply:
Any is used at only this place https://github.com/oppia/oppia/blob/aaa1987fc06a5eee127ba41dc3fc8b4cfaf0af4c/core/domain/suggestion_registry.py#L311
 and this is already addressed.

Reply:
Er, I'm looking at that justification and it seems a bit off. What exactly prevents BaseSuggestion from using strict typing? The whole point is to use strict typing so saying "we use Any because we don't use strict typing" doesn't really explain anything.

Reply:
I'm not sure if you have actually opened the issue that is mentioned in the TODO because to cover all the info in just a code comment is hard.

Reply:
You don't need to write everything that is on the issue here. But try to summarize the actual problem and write that here; what you have written here is not clear.

Reply:
Comment updated PTAL!